src/components/user/SignUpForm.js

```diff
@@ -22,88 +20,92 @@ const messages = defineMessages({
   password: { id: 'login.form.password' },
   passConfirmation: { id: 'signup.form.passconfirmation' },
   gender: { id: 'signup.gender' },
-  genderSelect: { id: 'signup.genderSelect' }
+  genderSelect: { id: 'signup.genderSelect' },
+  male: { id: 'gender.male' },
+  female: { id: 'gender.female' },
+  other: { id: 'gender.other' }
 });
 
-const options = [
-  { value: 'male', label: 'Male' },
-  { value: 'female', label: 'Female' },
-  { value: 'other', label: 'Other' }
-];
+const signupForm = ({ error, handleSubmit, submitting, intl }) => {
+  const options = [
```

Comment:
I see you repeated this. Can you move this to a constants file? You may not be able to use `intl` there, but you can have the ids of the messages and use that after in the view to run intl on them.